### DISCUSSION TITLE
Please help with configuration for single node

### DISCUSSION THREAD

**Discussion by GrygorievEvgeniy** [autobase-tech/autobase#723] [Q&A]
How I can change config for single node in cluster.
I change /etc/patroni/patroni.yml in one of the nodes, but after restart nothing changes in config files.

**Comment by vitabaks** [comment]
Use `config_pgcluster.yml `playbook, details [here](https://github.com/vitabaks/postgresql_cluster#changing-postgresql-configuration-parameters)

Or `patronictl edit-config`

**Reply by GrygorievEvgeniy** [reply]
Or patronictl edit-config - This is not global change for cluster ?

**Reply by vitabaks** [reply]
global.

For single node only use patroni.yml 

`postgresql.parameters` section

**Reply by GrygorievEvgeniy** [reply]
And this is start of my topic. I change /etc/patroni/patroni.yml in one of the nodes, but after restart nothing changes in config files.

**Reply by vitabaks** [reply]
use psql and `show <param_name>;` command to check option.

**Reply by GrygorievEvgeniy** [reply]
I see this in postgres_exporter and prometheus, on grafana dashboards

**Reply by vitabaks** [reply]
It's strange because:

> These options are defined in the configuration file and take precedence over dynamic configuration.

https://patroni.readthedocs.io/en/latest/patroni_configuration.html

Try use `ALTER SYSTEM SET ...;` `select pg_reload_conf();` on specific node.
